client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/services/SecurityService.kt

```diff
@@ -3,14 +3,23 @@ package uk.ac.ebi.biostd.client.cli.services
 import uk.ac.ebi.biostd.client.cli.dto.PermissionRequest
 
 internal class SecurityService {
-    fun grantPermission(request: PermissionRequest) = performRequest { permission(request) }
+    fun grantCollectionPermission(request: PermissionRequest) =
```

Comment:
wondering if this really is a collection, basically we give permission to access No, like a submission I am wondering if command, documentaiton and this method name should reflect that.